snowshu/core/replica/replica_factory.py

```diff
@@ -100,6 +102,14 @@ def _execute(self,
                 self.config.target_profile.adapter.create_function_if_available(
                     function, relations)
             logger.info('Emulation functions applied.')
+
+            logger.info('Copying replica data to shared location...')
+            status_message = self.config.target_profile.adapter.copy_replica_data()
+            if status_message[0] != 0:
+                message = (f'Failed to execute copy command: {status_message[1]}')
+                logger.error(message)
+                raise UnableToExecuteCopyReplicaCommand(message)
+
             self.config.target_profile.adapter.finalize_replica()
```

Comment:
Do we want to clean up the docker volume once we're done so it isn't taking up space on the host system as part of this? Or leave that until later work when we know exactly where the volume will be used?